tests/nlu/featurizers/test_convert_featurizer.py

```diff
@@ -83,6 +87,8 @@ def test_convert_featurizer_train(component_builder):
         ("ńöñàśçií", "ńöñàśçií"),
     ],
 )
+@pytest.mark.linux
+@pytest.mark.darwin
```

Comment:
optional: Having the `linux` and `darwin` marker implicitly means that the tests don't run on Windows. I wonder if turning it around (e.g. having `@pytest.mark.skip_on_windows`) would make this more explicit